static/src/javascripts/projects/common/views/svgs.js

```diff
@@ -1,5 +1,10 @@
 // @flow
-// Include any images needed in templates here.
+
+// DEPRECATED: please don't use this file to inject SVGs into your markup
```

Comment:
@SiAdcock could you update our docs as well?
 https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/04-inline-svgs.md#javascript-templates